code/datums/ai/movement/ai_movement_jps.dm

```diff
@@ -2,7 +2,7 @@
  * This movement datum represents smart-pathing
  */
 /datum/ai_movement/jps
-	max_pathing_attempts = 20
```

Comment:
if no one noticed, why not leave it

Reply:
XD because having bots stand there looking at the same wall for ages is bad for bussiness